wrapper/Mathcad/README.md

```
@@ -54,39 +54,51 @@ Make the Build for Mathcad Prime (any version above 3.0)
              -DIF97_PRIME_ROOT="C:/Program Files/PTC/Mathcad Prime 10.0.0.0"  
              -G "Visual Studio 17 2022" -A x64 
-             -DCMAKE_VERBOSE_MAKEFILE=ON 
+             -DCMAKE_VERBOSE_MAKEFILE=ON
+    ```
 	     
-	> Insert your version of Mathcad Prime in place of "10.0.0.0".  
-    > Insert your version of Visual Studio for the -G option.  
-	> Note that Mathcad Prime is 64-bit and requires the `-A x64` switch on this command.  
-    > Prior to VS 2017, use something like: `-G "Visual Studio 14 2015 Win64`
+	- Insert your version of Mathcad Prime in place of "10.0.0.0" (or omit).  
+    - Insert your version of Visual Studio for the `-G` option.  
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 4

(MD005, list-indent)